ngraph/frontend/ir/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ install(TARGETS ${TARGET_NAME} EXPORT ngraphTargets
         ARCHIVE DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph
         LIBRARY DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph)
 
-install(DIRECTORY ${${TARGET_NAME}_INCLUDE_DIR}/ir_frontend
+install(DIRECTORY ${${TARGET_NAME}_INCLUDE_DIR}/ir_ngraph_frontend
```

Comment:
`_ngraph` is not needed here.

Secondly, do we really need IR FE for end users?

Reply:
@slyalin correct me if Im wrong but all FE can be used by user directly.

Reply:
> _ngraph is not needed 

done.